src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -215,7 +215,7 @@
     color: #FFFFFF;
 }
 
-.tooltip__button--manage {
+.tooltip__button--secondary {
```

Comment:
Just renaming it to sound more "reusable", just because we have the same scenario (manage password + don't show again) kind of like a secondary button.

Reply:
Nice.